releasenotes/notes/cleanup-rb-experiment-f17b6e674ae4e473.yaml

```diff
@@ -0,0 +1,38 @@
+---
+features:
+  - |
+    A helper funciton :func:`lookup_epg_ratio` has been added. 
```

Comment:
If these are made internal helper functions, they dont need to be added to the release notes